src/rpc/core_rpc_server.cpp

```diff
@@ -378,14 +380,6 @@ namespace cryptonote
 
     m_net_server.get_config_object().m_max_content_length = MAX_RPC_CONTENT_LENGTH;
 
-    if (store_ssl_key && inited)
```

Comment:
SSL keys/certs are now stored in the call to `epee::http_server_impl_base<core_rpc_server, connection_context>::init` above so this block isn't needed.